build.sbt

```diff
@@ -800,7 +800,7 @@ lazy val complianceTests = projectMatrix
         Dependencies.Circe.parser.value,
         Dependencies.Http4s.circe.value,
         Dependencies.Http4s.client.value,
-        Dependencies.Weaver.cats.value % Test,
+        Dependencies.Weaver.cats.value ,
```

Comment:
the `compliance-test` module needs to remains test-framework agnostic, the weaver-specific abstraction should go to the `tests` module instead